tests/test_batch.py

```diff
@@ -31,13 +32,21 @@ def test_make_batches_mvp_v1(nwp_data, pv_yields_and_systems):
         )
 
 
-def test_make_batches_mvp_v2(nwp_data, pv_yields_and_systems):
+def test_make_batches_mvp_v2(nwp_data, pv_yields_and_systems, sat_data, hrv_sat_data):
 
     with tempfile.TemporaryDirectory() as temp_dir:
         # save nwp data
         nwp_path = f"{temp_dir}/unittest.netcdf"
         nwp_data.to_netcdf(nwp_path, engine="h5netcdf")
         os.environ["NWP_PATH"] = nwp_path
+        hrv_sat_path = f"{temp_dir}/hrv_sat_unittest.zarr.zip"
+        with zarr.ZipStore(hrv_sat_path) as store:
+            hrv_sat_data.to_zarr(store, compute=True)
+        os.environ["HRV_SAT_PATH"] = hrv_sat_path
+        sat_path = f"{temp_dir}/sat_unittest.zarr.zip"
+        with zarr.ZipStore(sat_path) as store:
+            sat_data.to_zarr(store, compute=True)
+        os.environ["SAT_PATH"] = sat_path
 
         make_batches(
             config_filename="nowcasting_forecast/config/mvp_v2.yaml", temporary_dir=temp_dir
```

Comment:
small thing below here
Could you check that satellite and hrv batches have been made - just to be double sure

Reply:
Okay, sure